examples/svm_batch.py

```diff
@@ -60,6 +60,16 @@ def main(readcsv=read_csv, method='defaultDense'):
 
 if __name__ == "__main__":
     (predict_result, plabels) = main()
+    """
+    The return value of SVM algorithm is float data, inorder to be consistent with plabels read from csv
+    We need to binarize the result
+    """
+    for i in range(len(predict_result.prediction)):
```

Comment:
I think better return decision function and result of classification. See example for GPU: https://github.com/IntelPython/daal4py/blob/master/examples/sycl/svm_batch.py#L104

Reply:
yeah, the suggestion is nice and I have updated the code and submitted a new pull request